src/less/workingfiles/submit.less

```diff
@@ -250,17 +251,21 @@ td {
     background-color: @super-light-gray;
 }
 
+
 #queue-table th:first-child {
-    width: 22%;
+
 }
 
+div .table-cell-div{
+    overflow: scroll;
```

Comment:
I think it looks better without the scroll bars. On my machine (chrome/windows) it looks good anyway so not sure if it's even needed. Here is how it looks with line 260 removed:
![no-scroll-bars](https://cloud.githubusercontent.com/assets/6011225/23310161/d3ecd994-fab1-11e6-8593-a1cf07fedd1e.PNG)